README.md

```diff
@@ -5,4 +5,5 @@
 ## Implementations
 
 - [dif](https://github.com/decentralized-identity/lds-ecdsa-secp256k1-2019.js)
-- [digitalbazaar](https://github.com/digitalbazaar/secp256k1-key-pair)
\ No newline at end of file
+- [digitalbazaar](https://github.com/digitalbazaar/secp256k1-key-pair)
+- [spruce](https://docs.rs/ssi/0.2.0/ssi/ldp/struct.EcdsaSecp256k1Signature2019.html)
```

Comment:
```suggestion
- [spruce](https://docs.rs/ssi/0.2.0/ssi/ldp/struct.EcdsaSecp256k1Signature2019.html)
- [danubetech](https://github.com/weboftrustinfo/ld-signatures-java)
```

Could we also just add our implementation in the same PR? If that's bad idea then I'd also be happy to just create my own PR after yours gets merged.

Reply:
Sure! Adding...